libs/net/dal/Services/AVOverviewTemplateService.cs

```diff
@@ -91,7 +91,6 @@ public override AVOverviewTemplate Update(AVOverviewTemplate entity)
         });
         entity.Sections.ForEach(section =>
         {
-            section.Template = entity;
```

Comment:
Have you tested adding and updating sections?  This was most likely needed when creating a new section an linking it to an existing template.

Reply:
Yes and it seems to be working fine so far.